topology/University of Chicago/UChicago/UChicago_OSGConnect.yaml

```diff
@@ -13,63 +13,330 @@ Resources:
     ContactLists:
       Administrative Contact:
         Primary:
+          Name: Judith Stephen
+          ID: a418fbc5dd33637bba264c01d84d52dd317f2813
+        Secondary:
           Name: Lincoln Bryant
           ID: 0a22bab3de2d83d723811e3fb1ebca904e924a97
+      Security Contact:
+        Primary:
+          Name: Judith Stephen
+          ID: a418fbc5dd33637bba264c01d84d52dd317f2813
         Secondary:
+          Name: Lincoln Bryant
+          ID: 0a22bab3de2d83d723811e3fb1ebca904e924a97
+    FQDN: login04.osgconnect.net
+    DN: /DC=org/DC=incommon/C=US/ST=IL/L=Chicago/O=University of Chicago/OU=IT Services - Self Enrollment/CN=condor-flocking-cert.grid.uchicago.edu
```

Comment:
Is this the right DN? Shouldn't the CN match the hostname?

Reply:
A lot of the flocking hosts appear to share a cert. I imagine that it's got a lot of SANs but @rynge or @LincolnBryant would know how this actually works.